frontend/src/views/admin/CategoryAdd.jsx

```diff
@@ -1,13 +1,39 @@
 import React from 'react'
 import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import axiosInstance from '../../api/api'
 import Divider from '../../components/Divider'
+import { AppContext } from '../../context/AppContext'
 
 const CategoryAdd = () => {
+    const context = React.useContext(AppContext)
+    const navigate = useNavigate();
+
+    const [currentUser, setCurrentUser] = context.user
     const [lessonTitle, setLessonTitle] = useState("")
-    const [lessonBody, setLessonBody] = useState("")
+    const [lessonSlug, setLessonSlug] = useState("")
+    const [lessonDescription, setLessonDescription] = useState("")
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        const formData = new FormData()
 
-    const handleSubmit = () => {
-        alert("Added Category!") 
+        formData.append('title', lessonTitle)
+        formData.append('slug', lessonSlug)
+        formData.append('description', lessonDescription)
+        axiosInstance.post("categories",
+            formData,
+            {
+                headers: {
+                    'Authorization': `Bearer ${currentUser.token}`,
+                }
```

Comment:
Add the `headers.Authorization` in the `axiosInstance` and apply the cookies that we discussed earlier